packages/espanso-dropdown-list-template/_manifest.yml

```diff
@@ -0,0 +1,7 @@
+name: espanso-dropdown-list-template
+version: 1.0.0
+author: Omar Elmenawi
+description: Espanso shortcut for creating structured and modular text replacements with custom list options.
+repository: https://github.com/menawi/espanso-dropdown-list-template
+
+
```

Comment:
Some `tags:` would be helpful.

Reply:
Added tags and removed repository so the package is more in line with other submissions